gimme_aws_creds/config.py

```diff
@@ -210,7 +210,7 @@ def update_config_file(self):
             'write_aws_creds': '',
             'cred_profile': 'role',
             'okta_username': '',
-            'app_url': '',
+			'app_url': '',
```

Comment:
Should this be indented like this?

Reply:
whoops! just an indentation problem :)
Fixed in the next commit.